fixed duplicate files

shreeya13garg opened the pull request:
**Fixes issue:**
<!-- [Mention the issue number it fixes or add the details of the changes if it doesn't has a specific issue. -->
Duplicate code for linked list

**Changes:**
<!-- Add here what changes were made in this pull request. -->
Deleted the directory file singly_linked_list from data_structures/src/list/ path.

<!-- Make sure to look at the Style Guide for your language in guides/coding_style/language_name:

     https://github.com/OpenGenus/cosmos/tree/master/guides/coding_style

     Note: A coding style guide may not exist for your language, since this is still in beta.
-->

<!-- Make sure to look at the Documentation Style Guide in guides/documentation.md:

     https://github.com/OpenGenus/cosmos/blob/master/guides/documentation_guide.md

     The document style guide may not apply for your algorithm category, you must also look at specified guide under all of the directory in the category, e.g., for project euler:

     https://github.com/OpenGenus/cosmos/blob/master/code/online_challenges/src/project_euler/documentation_guide.md
-->

arnavb commented:
@shreeya13garg I think [these](https://github.com/OpenGenus/cosmos/tree/master/code/data_structures/src/linked_list) are the duplicate files, not the ones you deleted. There are more files in the folder you deleted.

shreeya13garg commented:
Oh i thought either of the duplicate files can be removed.

On Wed, 25 Mar 2020 at 3:23 AM, Arnav Borborah <notifications@github.com>
wrote:

> @shreeya13garg <https://github.com/shreeya13garg> I think these
> <https://github.com/OpenGenus/cosmos/tree/master/code/data_structures/src/linked_list>
> are the duplicate files, not the ones you deleted. There are more files in
> the folder you deleted.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenGenus/cosmos/pull/4724#issuecomment-603524082>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AMLBWJD4GVWMJJLBF3HHNQ3RJETT7ANCNFSM4LSNA55Q>
> .
>

arnavb commented:
Since there are less files in the directory I picked, I would delete those.

shreeya13garg commented:
I did what you asked to do. Thanks for your help. Please look and let me
know if it's alright or not.

On Wed, Mar 25, 2020 at 3:26 AM Arnav Borborah <notifications@github.com>
wrote:

> Since there are less files in the directory I picked, I would delete those.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenGenus/cosmos/pull/4724#issuecomment-603525542>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AMLBWJFRQ4INKNLV6KQJP73RJEUA5ANCNFSM4LSNA55Q>
> .
>

arnavb commented:
@shreeya13garg No the wrong directory is still deleted. I wanted you to delete this folder: https://github.com/OpenGenus/cosmos/tree/master/code/data_structures/src/linked_list.

shreeya13garg commented:
Should I delete this folder?

On Wed, 25 Mar 2020 at 6:30 PM, Arnav Borborah <notifications@github.com>
wrote:

> @shreeya13garg <https://github.com/shreeya13garg> No the wrong directory
> is still deleted. I wanted you to delete this folder:
> https://github.com/OpenGenus/cosmos/tree/master/code/data_structures/src/linked_list
> .
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenGenus/cosmos/pull/4724#issuecomment-603825098>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AMLBWJFJWNLOOIFRMX5G4NTRJH57ZANCNFSM4LSNA55Q>
> .
>

arnavb commented:
Yes, delete the folder I linked to and revert the commit where you delete the folder you have now.

shreeya13garg commented:
But haven't your merged my commit as I got a mail from you recently that
nice work and the mail mentioned that my work was merged.

On Thu, Mar 26, 2020 at 1:34 AM Arnav Borborah <notifications@github.com>
wrote:

> Yes, delete the folder I linked to and revert the commit where you delete
> the folder you have now.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenGenus/cosmos/pull/4724#issuecomment-604057061>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AMLBWJEW46F7XND75N5EUUTRJJPTHANCNFSM4LSNA55Q>
> .
>

shreeya13garg commented:
One more thing this link-
https://github.com/OpenGenus/cosmos/tree/master/code/data_structures/src/linked_list
is
showing 404 error and is not opening. Isnt this link which is referring to
the folder I deleted previously.
I previously deleted files from
cosmos/code/data_structures/src/linked_list. Last time I deleted the
linked_list folder from src do you want me to that again?

On Thu, Mar 26, 2020 at 10:20 AM Shreeya Garg <shreeya18415@iiitd.ac.in>
wrote:

> But haven't your merged my commit as I got a mail from you recently that
> nice work and the mail mentioned that my work was merged.
>
> On Thu, Mar 26, 2020 at 1:34 AM Arnav Borborah <notifications@github.com>
> wrote:
>
>> Yes, delete the folder I linked to and revert the commit where you delete
>> the folder you have now.
>>
>> —
>> You are receiving this because you were mentioned.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/OpenGenus/cosmos/pull/4724#issuecomment-604057061>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AMLBWJEW46F7XND75N5EUUTRJJPTHANCNFSM4LSNA55Q>
>> .
>>
>

arnavb commented:
! @shreeya13garg Thanks for catching my mistake. I forgot I had merged yours. Well that means the mistake is already fixed. Your other PRs for this can be closed.

shreeya13garg commented:
So this issue is solved na..Can i move to next issue now?

On Thu, 26 Mar 2020 at 7:21 PM, Arnav Borborah <notifications@github.com>
wrote:

> ! @shreeya13garg <https://github.com/shreeya13garg> Thanks for catching
> my mistake. I forgot I had merged yours. Well that means the mistake is
> already fixed. Your other PRs for this can be closed.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenGenus/cosmos/pull/4724#issuecomment-604442547>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AMLBWJD6D36H7BZV5VPMVRDRJNMWFANCNFSM4LSNA55Q>
> .
>

shreeya13garg commented:
I want to know the path for codes of LCM code in python, Hill_cipher
algorithm, and Merging K sorted arrays.
I have attached the codes for the following. I wanted to know the path for
these codes so that I can add them.

arnavb commented:
LCM should go in mathematical algorithms (there should already be a folder and I think it's already done). Hill cipher in cryptography. Merging k sorted arrays in unclassified.

shreeya13garg commented:
Okay I will do both of it.

On Wed, Apr 1, 2020 at 11:42 PM Arnav Borborah <notifications@github.com>
wrote:

> LCM should go in mathematical algorithms (there should already be a folder
> and I think it's already done). Hill cipher in cryptography. Merging k
> sorted arrays in unclassified.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenGenus/cosmos/pull/4724#issuecomment-607409548>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AMLBWJDZBM6NDCIK2PU37FTRKN7Z3ANCNFSM4LSNA55Q>
> .
>
